samples/excel/16-custom-functions/basic-function.yaml

```diff
@@ -0,0 +1,22 @@
+order: 1
+id: excel-custom-functions-basic
+name: Basic custom function
+description: A simple custom function that calculates the volume of a sphere.
```

Comment:
In the other Excel snippets, we've had description naming patterns of "Verbs X." Like "Creates a few different geometric shapes and deletes them from the worksheet." I'm not sure that pattern makes sense for CFs, but something to consider.

Reply:
We could just say "Calculate the volume of a sphere" since the title already have "custom function" in it.

Reply:
Fixed!